### PR TITLE
Added @types/geojson to devDependencies

### PR DESCRIPTION
**Pull request by gcoro**
### Does any other open PR do the same thing?

No

### What issue is this PR fixing?

This is the fix for a review on PR #3331 

### How did you test this PR?

I tested it by running `tsc`. The reported error doesn't show up anymore

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "@babel/core": "^7.4.3",
     "@babel/runtime": "^7.4.3",
     "@react-native-community/eslint-config": "0.0.5",
+    "@types/geojson": "^7946.0.7",
```
Comment (SimenB) [inline package.json:43]: It needs to be in normal `dependencies`, otherwise dependents would still need to install it

**Comment by rborn** [comment]
@gcoro 
could you please update yarn.lock so maybe we make Travis build? 

I see this error:
```
yarn install v1.15.2
[1/4] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Thanks

**Comment by gcoro** [comment]
@rborn done :)

**Comment by rborn** [comment]
@SimenB do you wanna give it a shot now?

**Comment by SimenB** [comment]
```json
"resolutions": {
  "react-native-maps": "gcoro/react-native-maps#bb4d49e831df3bea40da834f58694df107edf953"
}
```

fixed the type error 👍

**Comment by rborn** [comment]
@gcoro thank you 😻

**Pull request merged**